confgenerator/files.go

```diff
@@ -16,6 +16,10 @@ func GenerateFiles(input, service, logsDir, stateDir, outDir string) error {
 	if err != nil {
 		return err
 	}
+	return GenerateFilesFromUnifiedConfig(uc, service, logsDir, stateDir, outDir)
+}
+
+func GenerateFilesFromUnifiedConfig(uc *UnifiedConfig, service, logsDir, stateDir, outDir string) error {
```

Comment:
This should be a method (and it can keep the name `GenerateFiles` in that case)

Reply:
Done, thanks.